lib/reek/cli/command/commandable.rb

```diff
@@ -0,0 +1,35 @@
+module Reek
+  module CLI
+    module Command
+      #
+      # Base class for all commands
+      #
+      # :reek:ModuleInitialize
```

Comment:
Hm, i'm not so sure. Perhaps initialize belongs in a Command superclass, while the scanner code could be in a module, or perhaps even better in a class that the commands delegate to.